Source/Install/MediaInfo_GUI_Windows_EvergreenStub.nsi

```diff
@@ -0,0 +1,106 @@
+#NSIS: encoding=UTF-8
+RequestExecutionLevel admin
+
+; Some defines
+!define PRODUCT_NAME "MediaInfo"
+!define PRODUCT_PUBLISHER "MediaArea.net"
+!define PRODUCT_VERSION "2024.12.9"
```

Comment:
I understand the proposal here but I think I prefer to keep consistency and use 24.12 here.

Reply:
Done.